lib/testobject.js

```diff
@@ -207,11 +212,25 @@ TestObject.fetchAppium = async function (driverName, driverUrl) {
     proc.start();
   });
   // Zip it and return it
+  const pathToAppiumCopy = path.resolve(tempdir, 'appium');
   const pathToZip = path.resolve(tempdir, 'appium.zip');
-  logger.debug(`Writing zip file to ${pathToZip}`);
-  await exec('zip', ['-r', 'appium.zip', 'appium/'], {cwd: tempdir});
-  await fs.rimraf(path.resolve(tempdir, 'appium'));
+
+  //await zip(pathToZip, [pathToAppiumCopy]);
+  const pathToBestZip = path.resolve(process.env.PWD, 'node_modules', '.bin', 'bestzip');
```

Comment:
shouldn't we check if the binary exists first?

Reply:
Yeah I suppose that we could do that. It would produce a more helpful error message if it wasn't found.